test/components/contour/contour_test.go

```diff
@@ -42,14 +42,13 @@ func TestContourDeployment(t *testing.T) {
 	t.Parallel()
 	namespace := "projectcontour"
 	deployment := "contour"
-	replicas := 2
 
 	client, err := testutil.CreateKubeClient(t)
 	if err != nil {
 		t.Errorf("could not create Kubernetes client: %v", err)
 	}
 	t.Log("got kubernetes client")
 
-	testutil.WaitForDeployment(t, client, namespace, deployment, replicas, time.Second*5, time.Minute*5)
-	t.Logf("Found required replicas: %d", replicas)
+	testutil.WaitForDeployment(t, client, namespace, deployment, time.Second*5, time.Minute*5)
+	t.Logf("Found required replicas")
```

Comment:
This line can be moved to the `WaitForDeployment` function, to avoid duplicating it so many times. And to provide consistent output.